examples/phi2/phi2.py

```diff
@@ -59,9 +59,17 @@ def get_args(raw_args):
     parser.add_argument(
         "--model_type",
```

Comment:
What if `model_type` and `finetune_method` are None?

Reply:
fixed by raising error.